e2e/test-specs.yml

```diff
@@ -0,0 +1,74 @@
+description: |
+  End-to-end tests for k8s integration
+
+custom_test_key: k8s.clusterName
+
+scenarios:
+  - description: |
+      This scenario will verify that metrics from a k8s Cluster are correctly collected.
+    before:
+      - helm upgrade --install ${SCENARIO_TAG}-resources -n nr-${SCENARIO_TAG} --create-namespace ./charts/e2e-resources
+      - helm upgrade --install ${SCENARIO_TAG} -n nr-${SCENARIO_TAG} --create-namespace newrelic/nri-bundle --values e2e-values.yml --set global.licenseKey=${LICENSE_KEY} --set global.cluster=${SCENARIO_TAG}
```

Comment:
Where does `SCENARIO_TAG` come from? It would be great if we could include the short commit hash in it.

Reply:
it is already the truncated  COMMIT_SHA=${{ github.sha }}


```
 "e2e-" + r.commitSha[:7] + "-" + string(b)
```